@navikt/ds-css/button.css

```
@@ -11,36 +11,32 @@
   border-style: solid;
   border-width: 2px;
   min-height: 48px;
+  min-width: 48px;
   align-items: center;
+  padding: calc(var(--navds-spacing-2)) var(--navds-spacing-2);
+}
 
-  /* Padding calculated based on 18px font for 44px tall button */
-  padding: calc(var(--navds-spacing-3) - 2px) var(--navds-spacing-3);
+.navds-button__inner {
+  display: flex;
+  align-items: center;
+  margin: auto;
+  gap: var(--navds-spacing-2);
 }
 
-.navds-button > svg {
+.navds-button svg {
   font-size: 1.5rem;
   width: 1.5rem;
   height: 1.5rem;
 }
```

Height og width settes til `1em` i ikonet, så det trengs vel ikke her?
```suggestion
.navds-button svg {
  font-size: 1.5rem;
}
```